### ISSUE TITLE
Request: Star-rating slider

### ISSUE THREAD

**Issue by NCrusher74** [kieranb662/Sliders-SwiftUI#7]
I've found a tutorial for making a 5-star rating where each star is a full star, but I'm having trouble adapting it to use half-star ratings, and I suspect what I actually need is a slider where the "bar" is a row of 5 stars with an invisible knob, and it fills as you drag across it.

I've attempted it myself based off the code for other customizable sliders, but I can't quite get it right.

Would you be willing to add something like this to your package?

**Comment by kieranb662** [comment]
Hi!

Thanks for reaching out. I don't think my library is the one for you. I
think what you want is something like the file I attached. It's a rough
sketch but it should get you started. The hard part will most likely be
when you convert the drag gestures translation to a rating value. I'm a bit
busy right now so I hope this helps

Best Wishes
Kieran

**Comment by NCrusher74** [comment]
Hi thanks so much for responding! I don't think the attachment made it, though.

**Comment by kieranb662** [comment]
Does this link work?

https://gist.github.com/kieranb662/4714bf3733d1dd5746a9f8d358b1715a#file-starratingslider-swift

**Comment by NCrusher74** [comment]
Yes, thank you! I will see what I can do from here! Thank you so much!
